pkg/drivers/common.go

```diff
@@ -122,6 +125,21 @@ func MakeDiskImage(d *drivers.BaseDriver, boot2dockerURL string, diskSize int) e
 	return nil
 }
 
+// KillAPIServerProc will kill an api server proc if it exists
+func KillAPIServerProc(runner command.Runner) error {
+	// first check if it exists
+	rr, err := runner.RunCmd(exec.Command("pgrep", "kube-apiserver"))
+	if err == nil { // this means we might have a running kube-apiserver
+		pid, err := strconv.Atoi(rr.Stdout.String())
+		if err == nil { // this means we have a valid pid
+			if _, err = runner.RunCmd(exec.Command("kill", "-9", string(pid))); err != nil {
```

Comment:
Can you clarify how it is possible for an apiserver pid to exist if the containers are stopped, or at least the scenario this function exists for?

I would rather not take this shot-gun approach, as it may kill unintended non-minikube processes.

(also, you could do pkill -9, but that's beside the point)